### DISCUSSION TITLE
How to retrieve record id

### DISCUSSION THREAD

**Discussion by sansan061100** [filamentphp/filament#9017] [General]
127.0.0.1:8000/admin/users/1/edit

I want to take the id 1

**Comment by sansan061100** [comment]
Solved 
```$id = request()->route()->parameter('record');```

**Reply by leandrocfe** [reply]
`$this->record->id` or `$this->getRecord()->id`

**Reply by sansan061100** [reply]
thanks

**Reply by iranzithierry** [reply]
how to do it in static unction

**Reply by leandrocfe** [reply]
`$record` ?

**Reply by iranzithierry** [reply]
Yeah
